src/components/formatting/ResponseAdherence.tsx

```diff
@@ -18,7 +18,7 @@ const ResponseAdherence: React.ElementType = () => {
             <tbody>
```

Comment:
@justinlewis @jsmethie could we also revert the changes to this file since they are unrelated?

Reply:
I reverted this but note that the change was because the HTML wasn't rendering correctly. One of you might want to change thse:

                <td>>=</td>
to

                <td>;&gt;=</td>